git-keeper.py

```diff
@@ -67,10 +79,11 @@ def main():
         repo_gpg = repo_tar + '.gpg'
         with open(repo_tar, 'rb') as f:
             status = gpg.encrypt_file(
-                f, recipients=RECIPIENTS,
+                f, recipients='1A826DCC093BFE3FDA89AA57F9E6D8B5C3E634CE',
```

Comment:
can you obtain the list of recipients dynamically? I believe you can do:
```
recipients=[k.fingerprint for k in gpg.list_keys()]
```
I haven't tested this though